python/ray/data/_internal/execution/streaming_executor_state.py

```diff
@@ -702,3 +706,16 @@ def _ranker(op):
         )
 
     return [_ranker(op) for op in ops]
+
+
+def _actor_info_summary_str(info: _ActorInfo) -> str:
+    total = info.running + info.pending + info.restarting
+    base = f"Actors: {total}"
+
+    if total == info.running:
+        return base
+    else:
+        return (
+            f"{base} (running={info.running}, restarting={info.restarting}, "
+            f"pending={info.pending})"
+        )
```

Comment:
implement this as `_ActorInfo.__str__`. 
So the logs in `scale_up/scale_down` can use it as well